cpg_pipes/providers/images.py

```diff
@@ -0,0 +1,49 @@
+"""
+Docker images used in the pipelines.
+"""
+import os
+
+
+AVAILABLE = {
```

Comment:
Would it make sense to source this from a JSON file (or similar), so it can be updated programmatically whenever a new image version is built?

Should this potentially be part of the TOML options blob?

Reply:
We currently have all CPG variables essentially built-in in analysis-runner code https://github.com/populationgenomics/analysis-runner/blob/d4441f6ffaf308d97c7207507bbc9b0a1431ddeb/server/util.py#L17-L30
Would it make sense to make a TOML config be the source of those? And we could add other stuff like docker names there as well.